packages/components/src/internal/components/domainproperties/SystemFields.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { FC, memo, useCallback, useState } from 'react';
+
+import classNames from 'classnames';
+
+import { Collapse } from 'react-bootstrap';
+
+import { Grid } from '../base/Grid';
+
+import { SystemField } from './models';
+import { EXPAND_TRANSITION } from './constants';
+
+interface Props {
+    systemFields: SystemField[];
+}
+
+export const SystemFields: FC<Props> = memo(({ systemFields }) => {
+    const [collapsed, setCollapsed] = useState<boolean>(false);
+
+    const onToggle = useCallback(() => {
+        setCollapsed(!collapsed);
+    }, [collapsed]);
+
+    return (
+        <>
+            <div className="domain-system-fields">
+                <div className="domain-system-fields-header">
+                    <div className="domain-system-fields-header__icon" onClick={onToggle}>
+                        <i className={classNames('fa fa-lg', collapsed ? 'fa-plus-square' : 'fa-minus-square')} />
+                    </div>
+                    <div className="domain-system-fields-header__text" onClick={onToggle}>
+                        Default System Fields
```

Comment:
I pulled down the fbs locally to take a look and the headers for the "Default System Fields" and the "Custom Fields" looks a little big. It seems odd to me that they are larger than the text for the panel title. Thoughts on moving the font-size down to about 18px?

Reply:
Hmmm, I waffled on this—I took a look at changing the font-size to 18px (and tweaking the whitespace heights to accommodate). My first impression was that making the text smaller does make all of the surrounding text 'samey' in size, and it's a little hard to intuitively know what to look at. When the header text is bigger, the eye flow feels a little more natural to me. The user also already knows they're inside of 'Fields' because it's expanded, so it's less important to have 'Default System Fields' smaller relative to 'Fields', but it is important to know the distinction between the grid-section and the domain-row-section.
All that said, I don't have a strong preference, but I'd lean towards the 24px!
![Screenshot 2023-02-07 at 2 06 12 PM](https://user-images.githubusercontent.com/18537731/217377479-b7165b6e-ee21-413d-84c1-ecd585945124.png)
![Screenshot 2023-02-07 at 2 06 07 PM](https://user-images.githubusercontent.com/18537731/217377503-95b44e74-19cd-4441-949d-e29f89c2d7b5.png)

Reply:
FWIW, my initial preference is for the smaller font size, mainly because I don't like things screaming at me and the jump upwards in font size seems unnatural to me.  But that's just my opinion.

Reply:
I prefer the smaller font size as well. @labkey-hannah @labkey-keegang thoughts?

Reply:
Let's use 18px so that it's not significantly bigger than the "Fields" heading.

Reply:
Cool, done! :)